### ISSUE TITLE
object destructuring and object literal notation behave differently

### ISSUE THREAD

**Issue by despairblue** [feross/standard#181]
```javascript
const {
  prop1,
  prop2,
} = obj
```

is allowed but:

```javascript
const obj = {
  prop1,
  prop2,
}
```

is a warning. Regard the trailing comma.

**Comment by feross** [comment]
I believe this is a bug in eslint. I filed an issue: https://github.com/eslint/eslint/issues/2911

**Comment by despairblue** [comment]
:+1:

**Comment by feross** [comment]
Bug fixed on eslint master.

**Comment by feross** [comment]
This is fixed in eslint 0.24.1. Run `npm update` to get the fix.

**Comment by despairblue** [comment]
:clap: ty :smiley:
